app/src/main/java/com/fstyle/structure_android/data/repository/UserRepository.java

```diff
@@ -37,196 +22,101 @@
 public class UserRepository
         implements UserDataSource.RemoteDataSource, UserDataSource.LocalDataSource {
 
-    private static final String SELECT_ALL_USER_QUERY =
-            String.format("SELECT * FROM %s", UserDbHelper.UserEntry.TABLE_NAME);
+    private static UserRepository INSTANCE;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName